<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCooperation</class>
 <widget class="QDialog" name="DialogCooperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cooperation Control</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>I vessel</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>101</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="LE_startxI"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_startyI"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_thetaI"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_deltaI"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_indicatorI"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>74</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>start_x (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>start_y (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>theta (deg)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>delta (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>indicator</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>191</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Common</string>
   </property>
   <widget class="QLineEdit" name="LE_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>99</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time (s)</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="PB_getcurrentposition">
       <property name="text">
        <string>current position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_setup">
       <property name="text">
        <string>setup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_reset">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>II vessel</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>101</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="LE_startxII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_startyII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_thetaII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_deltaII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_indicatorII"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>74</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>start_x (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>start_y (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>theta (deg)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>delta (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>indicator</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>III vessel</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>101</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLineEdit" name="LE_startxIII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_startyIII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_thetaIII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_deltaIII"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_indicatorIII"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>85</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>start_x (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>start_y (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>theta (deg)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>delta (m)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>indicator</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
